targets/TARGET_STM/TARGET_STM32F3/TARGET_STM32F303xE/TARGET_NUCLEO_F303RE/PinNames.h

```diff
@@ -137,6 +137,8 @@ typedef enum {
     LED3        = PA_5,
     LED4        = PA_5,
     USER_BUTTON = PC_13,
```

Comment:
This pin is specified as not existing, left the new button map in but could also pull it out since it's not really needed.